src/centralserver/admin-ui/src/views/Members/Member/Details/DeleteMemberDialog.vue

```diff
@@ -36,11 +36,9 @@
             </span>
           </v-card-title>
           <v-card-text class="pt-4" data-test="delete-member">
-            {{
-              $t('members.member.details.areYouSure1', {
-                member: member.member_name,
-              })
-            }}
+            {{ $t('members.member.details.confirmDelete1') }}
```

Comment:
I suggest to use component interpolation (`slots` syntax) approach like demonstrated [here](https://stackoverflow.com/questions/37872212/vue-i18n-with-html-in-named-params#answer-54749964).

Reply:
Fixed.